x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -197,6 +203,30 @@ export class AlertsClient<
     return { hits, total };
   }
 
+  public async untrackAlertIdByIndices(alertId: string, indices: string[]) {
```

Comment:
should we change the definition of the arguments here to have an object like 
```
{
 index_name_1: [alert_id_1, alert_id_2]
}
```

Reply:
we can remove all the code around getting the alert state and just do it with ruleId. That seems simpler and cleaner what do you think?

Reply:
I also forget the we should add the attribute `conflicts` to equal to `proceed`